charts/sn-platform/templates/grafana/grafana-pvc.yaml

```diff
@@ -0,0 +1,44 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+{{- if .Values.monitoring.grafana }}
+{{- if and .Values.volumes.persistence .Values.grafana.volumes.persistence }}
+{{- if not (or .Values.grafana.volumes.data.storageClass .Values.grafana.volumes.data.storageClassName) }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: "{{ template "pulsar.fullname" . }}-{{ .Values.grafana.component }}-{{ .Values.grafana.volumes.data.name }}"
```

Comment:
use  `"{{ template "pulsar.grafana.volumes.pvc.name" . }}"` for simplicity?

Reply:
yes, we can also fix in https://github.com/streamnative/charts/blob/master/charts/sn-platform/templates/prometheus/prometheus-pvc.yaml#L26

Reply:
updated in https://github.com/streamnative/charts/pull/857/commits/58ad5f5e4511420056e1835ed28426a3a24ea135